src/app/auth/user-info/user-info.component.spec.ts

```diff
@@ -2,19 +2,28 @@ import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { MatIconModule } from '@angular/material/icon';
 import { RouterTestingModule } from '@angular/router/testing';
 
+import { AuthService } from 'src/app/services/auth.service';
 import { UserInfoComponent } from './user-info.component';
 
 describe('UserInfoComponent', () => {
   let component: UserInfoComponent;
   let fixture: ComponentFixture<UserInfoComponent>;
+  let getUserInfoSpy: jasmine.Spy;
+  let testUserInfo: string = 'Den';
 
   beforeEach(async () => {
+    const authService = jasmine.createSpyObj('AuthService', ['getUserInfo']);
+    getUserInfoSpy = authService.getUserInfo.and.returnValue(testUserInfo);
```

Comment:
if you use spyOn only for mock data, you can simplify it 
providers: [{
provide: AuthService,
useValue: {
   getUserInfo: () => testUserInfo
}
}}